core/types/system_tx.go

```diff
@@ -0,0 +1,104 @@
+package types
+
+import (
+	"bytes"
+	"math/big"
+
+	"github.com/scroll-tech/go-ethereum/common"
+	"github.com/scroll-tech/go-ethereum/rlp"
+)
+
+type SystemTx struct {
+	ChainID *big.Int
+	To      *common.Address // system contract
+	Nonce   uint64          // nonce
```

Comment:
Do we bump `Nonce` normally for these txs? (Related discussion: https://www.notion.so/scrollzkp/L2-nonce-handling-for-L1MessageTx-draft-2d102df8cf1d43fe8a65e2afe1d112ca?pvs=4)